### ISSUE TITLE
What does the C. in Kent C. Dodds stand for?

### ISSUE THREAD

**Issue by nyrosmith** [kentcdodds/ama#94]

**Comment by kentcdodds** [comment]
Hi @nyrosmith! Thanks for the question. Honestly I'm surprised it's taken this long for anyone to ask on my AMA. People ask me in real life all the time :-)

It stands for Christopher. You didn't ask but I'll give you a backstory to why I prefer that it's included anywhere my full name is said or in print.

I started using the username `kentcdodds` places because even though I'm not aware of anyone who also has the name "Kent Dodds" I figured "Kent C. Dodds" would be even more rare and I'd have a better time avoiding issues with my username already being registered before I can sign up.

So when it came time to buy my first domain, I decided on `kentcdodds.com` because that made sense. I got the email `me@kentcdodds.com` and started using that.

Because of this, my friend @jakelingwall started calling me "Kent C. Dodds" all the time. We went to school together and worked together, so everyone else at school and work started calling me "Kent C. Dodds" too. I didn't mind, and after a while it started growing on me. In fact it did so much that when Josh James (Domo CEO and someone for whom I have much fear and respect) called me "Kent Dodds" I corrected him and said "No Josh, it's Kent **C.** Dodds" (this I did without thinking and felt pretty sheepish :sheep: for doing so).

So now I keep it and prefer it because I like it, and as @blesh said: "Everyone needs a gimmick" I guess the "C." is mine :-)

Thanks for asking!
